components/client/widgets/widget-selector.tsx

```diff
@@ -0,0 +1,82 @@
+"use client";
+
+import { Container } from "@uoguelph/react-components/container";
+import { useContext } from "react";
+import { SectionContext } from "@/components/client/section";
+import { AccordionWidget } from "@/components/client/widgets/accordions";
+import { ButtonSectionWidget } from "@/components/client/widgets/button-section";
+import { GeneralTextWidget } from "@/components/client/widgets/general-text";
+import { LinksWidget } from "@/components/client/widgets/links";
+import { MediaTextWidget } from "@/components/client/widgets/media-text";
+import { SectionWidget } from "@/components/client/widgets/section.js";
+import { StatisticsWidget } from "@/components/client/widgets/statistics";
+import { ImageOverlayWidget } from "@/components/client/widgets/image-overlay";
+import { StoryWidget } from "@/components/client/widgets/story";
+import { BlockWidget } from "@/components/client/widgets/block";
+import { TabsWidget } from "@/components/client/widgets/tabs";
+import { TestimonialSliderWidget } from "@/components/client/widgets/testimonial-slider";
+import { SocialMediaWidget } from "@/components/client/widgets/social-media";
+import { ProfileBlock } from "@/components/client/widgets/profile-block";
+import { ProfileCard } from "@/components/client/widgets/profile-card";
+import type { Widgets } from "@/data/drupal/widgets";
+
+export function WidgetSelector({ data }: { data: Widgets }) {
+  // If this widget is within a section, we don't want to render a container around it
+  const context = useContext(SectionContext);
+
+  // Some widgets need to span the full width of the page
+  const noWrapWidgets = ["ParagraphTestimonialSlider", "ParagraphImageOverlay", "ParagraphStoryWidget"];
```

Comment:
```suggestion
  const noWrapWidgets = ["ParagraphTestimonialSlider", "ParagraphImageOverlay", "ParagraphStoryWidget", "ParagraphProfileBlock"];
```

Because the Profile Listing widget is supposed to span the full width of the page: https://ugnext.netlify.app/review-profiles-csahs

Currently, it's getting cut off:

<img width="1606" height="547" alt="image" src="https://github.com/user-attachments/assets/02a57668-0315-46a3-9881-ab2573d4096e" />